packages/storybook8/stories/INTERNAL/ComplianceBanner/Docs.mdx

```diff
@@ -0,0 +1,13 @@
+import { Canvas, Meta } from '@storybook/blocks';
+import * as ComplianceBannerStories from './index.stories';
+import ComplianceBannerExampleText from '!!raw-loader!./snippets/ComplianceBanner.snippet.tsx';
+
+<Meta of={ComplianceBannerStories} />
+
+# Compliance Banner
+
+A component that displays banners to notify the user when call recording and transcription is enabled or disabled in a call.
+
+This component implements a state machine that tracks the changes to call recording and transcription state and shows the corresponding message.
+
+<Canvas of={ComplianceBannerStories.ComplianceBanner} source={{ code: ComplianceBannerExampleText }} />
```

Comment:
Should we have props in here?

Reply:
ComplianceBanner doesnt have a props table in it in storybook6 howver it does feel odd. Since its fairly quick to add it then lets just add it else we can leave it as a bug for someone else to add in later.

![image](https://github.com/user-attachments/assets/7768f280-8e48-43f7-9d62-d30a208e303b)

This is the bottom of the page and it doesnt have  props/argsTypes table